gradle-plugin/RELEASING.md

```diff
@@ -0,0 +1,33 @@
+# Release Steps
+
+To release the Linkage Checker Gradle plugin, run the following commands:
+
+```
+$ cd cloud-opensource-java
```

Comment:
as with the BOM, we should incorporate these commands into the release script to save us time

Reply:
Right. These are redundant now. Removed them.